src/shared/lib/geography/country-levels.js

```diff
@@ -51,43 +51,50 @@ export const getLocationData = async id => {
   const levelData = await getLevelData(level);
   const locationData = levelData[code];
 
-  assert(locationData, `Country Level data missing for: ${id}`);
+  if (!locationData) {
+    console.error(`Country Level data missing for: ${id}`);
+  }
 
   return locationData;
 };
 
 export const getFeature = async id => {
   const locationData = await getLocationData(id);
-  if (Array.isArray(locationData)) {
-    const features = await Promise.all(locationData.map(l => getFeature(l.countrylevel_id)));
-    return geography.combineFeatures(features);
-  }
-  if (locationData.geojson_path) {
-    const geojsonPath = path.join(countryLevelsDir, 'geojson', locationData.geojson_path);
-    const feature = await readJSON(geojsonPath);
-    return feature;
+  if (locationData) {
```

Comment:
I'd invert this for cleaner code. If not: return undefined, ...
Also, should we settle on null or undefined?